internal/events/pubsub/pubsub.go

```diff
@@ -6,9 +6,8 @@ import (
 	"regexp"
 	"time"
 
-	"cloud.google.com/go/functions/metadata"
```

Comment:
this was a bizarre thing to do...

Reply:
was it? I was just making structs out of structs 🤔